CMR_stimgeneration/Create_CMR_stim.m

```diff
@@ -62,21 +61,25 @@
 ERB_human_Hz = 24.7*(4.37*f_Hz/1000+1);  % from Moore and Glasberg (1983) [456 Hz at 4kHz]
 %  B.C.J. Moore and B.R. Glasberg, "Suggested formulae for calculating
 %  auditory-filter bandwidths and excitation patterns" Journal of the Acoustical Society of America 74: 750-753, 1983. 
-
-% Pick chin (1) or human (0)
-chin = 1;  % 0 = human
-if chin
-    BWnoise_Hz = ERB_chin_Hz;
-    fprintf('...Using chin ERBs\n')
-else % human 
-    BWnoise_Hz = ERB_human_Hz;
-    fprintf('...Using human ERBs\n')
+%% Menu Selection: Chin or Human
+% Pick chin (C) or human (H)
+subjectPrompt = '\nSubject: Chinchilla (C) OR Human (H) -> ';
+subject = input(subjectPrompt);
+if subject == 'C' || subject == 'c'
+   BWnoise_Hz = ERB_chin_Hz;
+   fprintf('...Using chin ERBs\n')
+   CMRcondition='CMRChin';
+else if subject == 'H' || subject == 'h' 
+        BWnoise_Hz = ERB_human_Hz;
+        fprintf('...Using human ERBs\n')
+        CMRcondition='CMRHuman';
+else
+    error('Please enter a valid character (C or H)');
+    end
 end
-
-%%%%%%%%%%%%%%%%%%% 
-% Adjust to find threshold
-%%%%%%%%%%%%%%%%%%%
-%% LATER - 
+% CMR condition predetermined from Menu option (CMRChin or CMRHuman)
```

Comment:
For tracking versions of the stim we use with the chins, we need to have some condition # in the name, like CMR1,2,3, ....  good to have chin & human in there too